spec/spec_helper.rb

```diff
@@ -1,5 +1,5 @@
-require "codeclimate-test-reporter"
-CodeClimate::TestReporter.start
+require "simplecov"
+SimpleCov.start
```

Comment:
Thanks for fixing that. Could you also add the following to the travis.yml so we still get the test coverage reporting with code climate?

```yml
after_success:
  - bundle exec codeclimate-test-reporter
```